packages/widget/tests/unit/components/transfer/fungible/fungible-token-transfer.test.ts

```diff
@@ -101,4 +112,71 @@ describe('Fungible token Transfer', function () {
 
     assert(sygmaAddressInput.address === '');
   });
+
+  it('should filter whitelisted networks and resources', async () => {
+    const whitelistedSourceNetworks = ['cronos'];
+    const whitelistedDestinationNetworks = ['sepolia'];
+    const whitelistedResources = ['ERC20LRTest'];
+    const connectedAddress = '0x95222290DD7278Aa3Ddd389Cc1E1d165CC4BAfe5';
+    const walletContext = await fixture<WalletContextProvider>(html`
+      <sygma-wallet-context-provider></sygma-wallet-context-provider>
+    `);
+
+    const fungibleTransfer = await fixture<FungibleTokenTransfer>(
+      html`<sygma-fungible-transfer
+        .whitelistedSourceNetworks=${whitelistedSourceNetworks}
+        .whitelistedDestinationNetworks=${whitelistedDestinationNetworks}
+        .whitelistedSourceResources=${whitelistedResources}
+        .environment=${Environment.TESTNET}
+      ></sygma-fungible-transfer>`,
+      { parentNode: walletContext }
+    );
+
+    walletContext.dispatchEvent(
+      new WalletUpdateEvent({
+        evmWallet: {
+          address: connectedAddress,
+          providerChainId: 11155111,
+          provider: getMockedEvmWallet().provider
+        }
+      })
+    );
+
+    // Set Source and Destination Networks
+    fungibleTransfer.transferController.onSourceNetworkSelected(cronosNetwork);
```

Comment:
Ain't you suppose to check before this that `fungibleTransfer.transferController.supportedSource(Destination)Networks˙is correctly whitelisted?